modules/auth_email/utils/code_management.ts

```diff
@@ -0,0 +1,92 @@
+import { RuntimeError, ScriptContext, Module } from "../module.gen.ts";
+
+const MAX_ATTEMPT_COUNT = 3;
+const EXPIRATION_TIME = 60 * 60 * 1000;
+
+
+
+export async function createVerification(ctx: ScriptContext, email: string) {
+	// Create verification
+	const code = Module.tokens.generateRandomCodeSecure("ABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789", 8);
```

Comment:
let's make this a separate token column that's indexed. we always need a secure way to look up data by an id that isn't sensitive.